### PR TITLE
addition of redux

### PR DESCRIPTION
**Pull request by endro42**

### CODE REVIEW

**Comment by nerdvibe** [comment]
Hey @tomploem ,

✨  Awesome, I'm really looking forward to have a look into this PR... Is this ready to be reviewed?

**Comment by endro42** [comment]
I will add a commit with the prettier extension so all files are prettified

**Comment by tomploem** [comment]
ready

File: src/actions/index.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import axios from "axios";
```
Comment (xunga) [inline src/actions/index.js]: Did you try to run this file with Prettier? The import is using double quotes instead single quote..

Comment (xunga) [reply src/actions/index.js]: Oh no, seems like I'm forcing it in my prettier, shame on me...

**Comment by nerdvibe** [comment]
@tomploem this PR looks awesome! Merging it....

**Welcome Redux!**

**Pull request merged**